### PR TITLE
[ENH] Implemented `pipeline_name` and `pipeline_version` query params

### LINKED ISSUE #113: Allow querying for pipeline name and version

### PR DESCRIPTION
**Pull request by rmanaem**
- Closes #113 

## Checklist
_This section is for the PR reviewer_

- [x] PR has an interpretable title with a prefix (`[ENH]`, `[FIX]`, `[REF]`, `[TST]`, `[CI]`, `[MNT]`, `[INF]`, `[MODEL]`, `[DOC]`) _(see our [Contributing Guidelines](https://neurobagel.org/contributing/CONTRIBUTING#pull-request-guidelines) for more info)_
- [x] PR has a label for the release changelog or `skip-release` (to be applied by maintainers only)
- [x] PR links to GitHub issue with mention `Closes #XXXX`
- [x] Tests pass
- [ ] Checks pass

For new features:
- [ ] Tests have been added

For bug fixes:
- [ ] There is at least one test that would fail under the original bug conditions.

### CODE REVIEW

File: app/api/crud.py [hunk 1]
```diff
@@ -67,6 +69,10 @@ async def get(
         Non-imaging assessment completed by subjects.
     image_modal : str
         Imaging modality of subject scans.
+    pipeline_name : str
+        Name of pipeline run on subject scans.
```
Comment (surchs) [inline app/api/crud.py:73]:
```suggestion
        Name of pipeline run on subject imaging data.
```

File: app/api/crud.py [hunk 2]
```diff
@@ -67,6 +69,10 @@ async def get(
...
+    pipeline_version : str
+        Version of pipeline run on subject scans.
```
Comment (surchs) [inline app/api/crud.py:75]:
```suggestion
        Version of pipeline run on subject imaging data.
```

**Review by surchs — APPROVED** [review]
Thanks @rmanaem, looks good to me. I guess there isn't much to test atm directly here, we'd need to look at e2e tests. 

:cook:

File: app/api/crud.py (hunk 1 above)
Comment (rmanaem) [reply app/api/crud.py:73]: In the interest of keeping things in line with what's in the node API I think we should stick with scans.

File: app/api/crud.py (hunk 2 above)
Comment (rmanaem) [reply app/api/crud.py:75]: Same as above

**Pull request merged**